extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/ForwardedServerRequestWrapper.java

```diff
@@ -0,0 +1,314 @@
+package io.quarkus.vertx.http.runtime;
+
+import java.util.Map;
+
+import javax.net.ssl.SSLPeerUnverifiedException;
+import javax.net.ssl.SSLSession;
+import javax.security.cert.X509Certificate;
+
+import io.vertx.core.Handler;
+import io.vertx.core.MultiMap;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.*;
+import io.vertx.core.net.NetSocket;
+import io.vertx.core.net.SocketAddress;
+
+class ForwardedServerRequestWrapper implements HttpServerRequest {
```

Comment:
@stuartwdouglas Hi Stuart, can it be of some help if the request info being replaced with this wrapper be contained as `HttpServerRequest` attribute ? So that it can be logged if needed etc ? Not really sure, just checking.
All looks good

Reply:
I don't really know if it is needed, generally you know what your proxies are. Maybe wait and see if anyone asks for it?

Reply:
@stuartwdouglas sorry, I keep missing the comments from Github...yes, makes sense